### ISSUE TITLE
Unable to add custom js to the file:// documents

### ISSUE THREAD

**Issue by yurivict** [hromadadan/customjs#1]
Getting this message in the red color:
It seems that this page cannot be modified with me..

**Comment by junkycoder** [comment]
Yes, sadly CJS was designed to modify pages based on their URL origin ie. domain name.
